action.yml

```diff
@@ -5,6 +5,8 @@ inputs:
     description: 'Relative path of the script in your repo.'
   O365_CLI_SCRIPT:
     description: 'Office 365 CLI Script'
+  IS_POWERSHELL:
+    description: 'Used only with O365_CLI_SCRIPT. Default is true. If false the assumption is the script will be shell script i.e. .sh.'
 runs:
   using: 'docker'
```

Comment:
Is that needed now after we swapped to JavaScript?

Reply:
Yes realized that after I submitted the PR. I have now removed it.